app/views/decidim/plans/plans/_linked_plans.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="card card--action card--list">
```

Comment:
I'm having trouble finding this file in core—are we overriding a piece of core functionality or is this a new view?

Reply:
Oh right, that was overriding a view in the plans module- should I take it out, given that we're not using the plans module anymore?

Reply:
I think it's fine to leave in, Shaun wants to incorporate plans into the workflow down the line. Just forgot plans weren't in core!